diff --git a/source/zxing/zxing/ZXing.h b/source/zxing/zxing/ZXing.h
index 70f1bd0..5cf4af7 100644
--- a/source/zxing/zxing/ZXing.h
+++ b/source/zxing/zxing/ZXing.h
@@ -49,17 +49,17 @@ inline float nan() {return std::numeric_limits<float>::quiet_NaN();}
 
 #else
 
-//#include <cmath>
+#include <cmath>
 
 #include <math.h>
 
 
 namespace zxing {
 inline bool isnan_z(float v) {
-    return isnan(v);
+	return std::isnan(v);
 }
 inline bool isnan_z(double v) {
-    return isnan(v);
+	return std::isnan(v);
 }
 inline float nan() {return std::numeric_limits<float>::quiet_NaN();}
 }
